src/Assignment.java

```diff
@@ -27,7 +27,7 @@ public int getTotalPoints() {
 	
 	// method to return the grade earned on the assignment
 	public double getGrade() {
-		return pointsEarned / totalPoints;
+		return ((double) pointsEarned / totalPoints) * 100;
```

Comment:
This change makes the function much easier to understand, as now you return the actual grade instead of a decimal.